main/main.cpp

```diff
@@ -3345,6 +3360,20 @@ bool Main::start() {
 		}
 	}
 
+	if (!generate_pot_path.is_empty()) {
+		Ref<GDScriptEditorTranslationParserPlugin> gdscript_translation_parser_plugin;
+		Ref<PackedSceneEditorTranslationParserPlugin> packed_scene_translation_parser_plugin;
+		gdscript_translation_parser_plugin.instantiate();
+		packed_scene_translation_parser_plugin.instantiate();
+		EditorTranslationParser::get_singleton()->add_parser(gdscript_translation_parser_plugin, EditorTranslationParser::STANDARD);
+		EditorTranslationParser::get_singleton()->add_parser(packed_scene_translation_parser_plugin, EditorTranslationParser::STANDARD);
+		POTGenerator::get_singleton()->generate_pot(generate_pot_path);
+		EditorTranslationParser::get_singleton()->remove_parser(packed_scene_translation_parser_plugin, EditorTranslationParser::STANDARD);
+		EditorTranslationParser::get_singleton()->remove_parser(gdscript_translation_parser_plugin, EditorTranslationParser::STANDARD);
+		OS::get_singleton()->set_exit_code(EXIT_SUCCESS);
```

Comment:
```suggestion
#ifdef MODULE_GDSCRIPT_ENABLED
		Ref<GDScriptEditorTranslationParserPlugin> gdscript_translation_parser_plugin;
		gdscript_translation_parser_plugin.instantiate();
		EditorTranslationParser::get_singleton()->add_parser(gdscript_translation_parser_plugin, EditorTranslationParser::STANDARD);
#endif // MODULE_GDSCRIPT_ENABLED
		Ref<PackedSceneEditorTranslationParserPlugin> packed_scene_translation_parser_plugin;
		packed_scene_translation_parser_plugin.instantiate();
		EditorTranslationParser::get_singleton()->add_parser(packed_scene_translation_parser_plugin, EditorTranslationParser::STANDARD);

		POTGenerator::get_singleton()->generate_pot(generate_pot_path);

		EditorTranslationParser::get_singleton()->remove_parser(packed_scene_translation_parser_plugin, EditorTranslationParser::STANDARD);
#ifdef MODULE_GDSCRIPT_ENABLED
		EditorTranslationParser::get_singleton()->remove_parser(gdscript_translation_parser_plugin, EditorTranslationParser::STANDARD);
#endif // MODULE_GDSCRIPT_ENABLED

		OS::get_singleton()->set_exit_code(EXIT_SUCCESS);
```
Modules need checking here too, and lets make this a bit less dense